lib/commands/submission.js

```diff
@@ -68,21 +74,31 @@ function exportSubmission(argv, problem, cb) {
     if (e) return cb(e);
     if (submissions.length === 0) return cb('no submissions?');
 
-    // find the latest accepted one
-    var submission = _.find(submissions, function(x) {
-      // TODO: select by lang?
+    // get obj list contain required filetype
+    var submissionInTargetType = _.filter(submissions, function(x) {
+      if (argv.lang === 'all') {
+        return true;
+      }
+      return argv.lang === x.lang;
```

Comment:
could use one line to do the same:

return argv.lang === 'all' || argv.lang === x.lang;